roles/confluent.test/upgrade_testing/confluent-kafka-kerberos-customcerts-rhel-upgrade-550-current.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+## variables
+
+SCENARIO_NAME=confluent-kafka-kerberos-customcerts-rhel
+
+## Checkout 5.5.0-post branch
+
+echo "Checking out 5.5.0-post branch"
+git checkout 5.5.0-post
+
+## Run Molecule Converge on 5.5.0-post
+
+echo "Running molecule converge"
+(cd ../ && molecule converge -s confluent-kafka-kerberos-customcerts-rhel)
+
+## Checkout 6.0.0-post branch
+
+echo "Checkout 6.0.0-post branch"
+git checkout upgrade_testing
```

Comment:
This is a bug, needs to checkout to an existing working directory, as upgrade_testing dir does not exist in older versions.